src/tests/ContentLoader.test.tsx

```diff
@@ -36,6 +40,13 @@ test('content is fetched and rendered on success', async () => {
   await screen.findByText('Test content')
 })
 
+test('variant content is fetched and rendered on success', async () => {
+  render(
+    <ContentLoader contentId='test-content'/>
+  )
+  await screen.findByText('Variant content')
+})
+
```

Comment:
Per [your comment](https://github.com/openstax/k12-apps-raise/pull/47#issuecomment-1272088665) this test case doesn't add any coverage / value, so we can rely on the playwright test for this change.